src/main/webapp/WEB-INF/tags/header.tag

```diff
@@ -8,8 +8,7 @@
 				<a id="logo-container" href="/" class="brand-logo center">Efeeder</a>
 				<div class="countdown right" style="font-size: 16pt" data-date=""></div>
                 <ul id="slide-out" class="side-nav" style="margin: 0;">
-                    <li><div class="userView">
-							<img class="background" src="/assets/img/background-image.jpg">
+                    <li><div class="userView" style="background-color: #000c47">

```

Comment:
Instead of using a specific style here (style="background-color: #000c47") could you see if it is possible to use one of the classes that materialize provides? take a look at here: http://materializecss.com/color.html . If not, ignore this comment.

Also, I think your tabs are being replaced with spaces, and I think we decided to use TABS instead of spaces.  Could you take a look at that?